packages/cyberstorm/src/components/Tooltip/Tooltip.tsx

```diff
@@ -24,7 +26,7 @@ export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
 }) => {
   return (
     <RadixTooltip.Root>
-      <RadixTooltip.Trigger>{children}</RadixTooltip.Trigger>
+      <RadixTooltip.Trigger asChild={asChild}>{children}</RadixTooltip.Trigger>
```

Comment:
are there some cases where asChild would actually be false? IF not maybe just hard code it and remove the prop, otherwise leave as is.

Reply:
this is fixed, resolve conversation when you've looked at the commit 👌